### ISSUE TITLE
bug: LocalNotification `on` repeats reminders every hour after first reminder

### ISSUE THREAD

**Issue by dotrub** [ionic-team/capacitor#2747]
# Bug Report

## Capacitor Version
`npx cap doctor` output:

  @capacitor/cli 1.5.0

  @capacitor/core 1.5.0

  @capacitor/android 1.5.0

  @capacitor/ios 2.0.0

## Affected Platform(s)

- [x] Android
- [ ] iOS
- [ ] Electron
- [ ] Web

## Current Behavior
Set up a LocalNotification. I want to set up daily repeating reminders at a time the user has set. The `on` option seems best suited, so I collect a time, extract the hours and minutes, and set those values as `on: { hour: ${time.hour}, minute: ${time.minute} }`. At the scheduled time, the reminder triggers; I tap it and do what it wanted me to. An hour later, when the minutes of the time match the `on` field again, I get another reminder, even though the hour shouldn't match. I continue to get reminders for every hour after the original.

Worth noting that I tried to see if it was because `hour` wasn't being respected, by setting my reminder to about an hour before now and waiting for the minutes to match. The reminder does not pop up in this case; this bug is only present after the reminder has triggered the first time.

## Expected Behavior
`on` should only trigger when all the fields match. If I set hour and minute, I expect the reminder to only fire when the hour and minute match.

## Sample Code or Sample Application Repo

Not much code to provide for this one. Here's most of the reminder object I'm setting:

```
const reminder = {
    id: 100,
    schedule: { 
        on: {
            hour: 12,
            minute: 30,
        },
    },
};
```

With this reminder, my reminder triggers at 12:30pm, and then at 1:30pm, 2:30pm, and so on...

## Reproduction Steps

Set a reminder like the one above and wait for it to trigger. Then wait an hour and see it trigger again.

## Other Technical Details

`npm --version` output: 6.9.0

`node --version` output: v10.16.3

**Comment by IT-MikeS** [comment]
I gave this a shot, and I wasn't able to reproduce this with a clean `ionic start myApp tabs --type=react`

**Comment by Ionitron** [comment]
It looks like this issue didn't get the information it needed, so I'll close it for now. If I made a mistake, sorry! I am just a bot.

Have a great day!
Ionitron 💙

**Comment by dwieeb** [comment]
There's an open PR that fixes this, so I'll reopen.

**Comment by jcesarmobile** [comment]
closing since the linked PR was merged. Should be fixed in 2.4.0
